drake/CMakeLists.txt

```diff
@@ -28,11 +28,6 @@ endif()
 set(POD_NAME drake)
 include(cmake/pods.cmake)
 
-# set up Java, use a target of 1.6
-find_package(Java REQUIRED)
-include(UseJava)
-set(CMAKE_JAVA_COMPILE_FLAGS ${CMAKE_JAVA_COMPILE_FLAGS} -source 6 -target 6)
-
```

Comment:
@jonathan-owens : Can you really remove this?  drake is built with external_project_add so it will not see any of the find Java stuff from the superbuild.   I think you will need to find java in the super build then pass down the information to the drake project so that it will find the same one.  But, I don't think you can just remove the find.